src/openrct2/peep/peep.c

```diff
@@ -9918,6 +9926,10 @@ static void get_ride_queue_end(sint16 *x, sint16 *y, sint16 *z){
 	rct_xy16 result = { 0, 0 };
 	rct_map_element *mapElement = map_get_first_element_at(*x / 32, *y / 32);
 
+	if (mapElement == NULL) {
+		return;
```

Comment:
https://github.com/OpenRCT2/OpenRCT2/blob/f85dc8c/src/openrct2/peep/peep.c#L10277

The pathfinding goal won't be updated. It would try proceeding to entrance in absence of queue.